pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -455,6 +456,11 @@ func (lbaas *CloudLb) ensureLoadBalancerNodes(lbID uint64, port corev1.ServicePo
 		}
 	}
 
+	if len(addNodes) > 25 {
```

Comment:
@ansalamdaniel - when is `ensureLoadBalancerNodes` called? If it is called often, won't this lead to unwanted processing in each iteration?

Reply:
`ensureLoadBalancerNodes` will be if there is change in the service object or if a node is added/removed. Updated the PR with different logic. Please review once.